src/qml/jsruntime/qv4global_p.h

```diff
@@ -114,15 +114,10 @@ inline double trunc(double d) { return d > 0 ? floor(d) : ceil(d); }
 
 // Do certain things depending on whether the JIT is enabled or disabled
 
-#ifdef V4_ENABLE_JIT
+#define V4_ENABLE_JIT
```

Comment:
Just thinking about the next person to read this code: I would move the define above the "Do certain things..." comment. And perhaps add a comment along the lines of "Enabling for all platforms on Sailfish OS".

Reply:
+1 and I'd suggest to take this as an opportunity to fix JB#56972 and then do something as natural as

```
#if defined(Q_OS_SAILFISH)
#define V4_ENABLE_JIT
#endif
```

Reply:
> Just thinking about the next person to read this code: I would move the define above the "Do certain things..." comment. And perhaps add a comment along the lines of "Enabling for all platforms on Sailfish OS".

I wouldn't do that since it might break on another arch on that Qt version. I'd prefer to only enable it on tested arches.

Reply:
The changes here are cherry-picked from upstream. Better not modify the commits without good reasons. If we need adjustments, they would be better as separate commits but all the better if those can be avoided.

Reply:
Correcting myself: the last commit is not cherry-pick. But anyhow moving these #defines would introduce extra delta.